backend/data/processors/manual_fixes/INFSFixes.py

```diff
@@ -77,27 +60,6 @@ def INFS2101():
     return "(INFSCH3964 || INFSCH3971) && (INFS1609 || INFS2609) && INFS2603) || (INFSB13554 && INFS2603)"
 
 
-def INFS2605():
-    """
-    "original": "Prerequisite: INFS1603 AND (INFS2609 or INFS1609).<br/>Students completing Computer Science degrees or majors (including BINF, COMP, or SENG) are excluded from this course.<br/><br/>",
-    "processed": "INFS1603 && (INFS2609 || INFS1609). Computer Science degrees || majors (&& BINF || COMP || SENG) are"
-    """
-    # Add exclusions to courseProcessed.json. Assumes that 'majors' includes
-    # honours programs
-    COURSES["INFS1609"]["exclusions"]["COMP#"] = 1
-    COURSES["INFS1609"]["exclusions"]["COMP?1"] = 1
-
-    return "INFS1603 && (INFS2609 || INFS1609)"
-
-
-def INFS_2822():
-    """
-    "original": "Pre-requisite: Any of the following: COMM1190, OR  INFS1603/COMM1822 OR INFS1609/INFS2609<br/><br/>",
-    "processed": "Any of the following: COMM1190 || (INFS1603 || COMM1822) || (INFS1609 || INFS2609)"
-    """
-    return "COMM1190 || (INFS1603 || COMM1822) || (INFS1609 || INFS2609)"
-
-
 def INFS_3202():
```

Comment:
Not modified in this PR but shouldn't `INFSB13554` be split into `(INFSB1 && 3554)`?